Sources/HaCWebsiteLib/Controllers/EventApiController.swift

```diff
@@ -0,0 +1,103 @@
+import Kitura
+import KituraNet
+import SwiftyJSON
+import Foundation
+import DotEnv
+
+struct EventApiController {
+  static var handler: RouterHandler = { request, response, next in
+    response.headers["Content-Type"] = "text/plain; charset=utf-8"
+    guard let parsedBody = request.body else {
```

Comment:
You've done an awesome job of figuring out all the boilerplate required. Hopefully we can abstract this all away in the future!